src/pyciemss/PetriNetODE/interfaces.py

```diff
@@ -267,7 +267,7 @@ def load_and_calibrate_and_sample_petri_model(
     else:
         return processed_samples
 
-
+@pyciemss_logging_wrappper
```

Comment:
Should the logging wrapper also be added to `load_and_calibrate_and_optimize_and_sample_petri_model`?